server/src/main/java/com/saralein/server/router/Router.java

```diff
@@ -72,4 +76,15 @@ private Controller routeUri(Request request) {
             return errorController.updateStatus(404);
         }
     }
+
+    private Controller routeFileRequest(Request request) {
+        if (requestingPartialContent(request)) {
+            return partialContentController;
+        }
+        return fileController;
+    }
```

Comment:
This is a little strange to me. Would this still be here after the static middleware PR is completed?

Reply:
Modified this so the FileController delegates partial content requests to the PartialContentController.